<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>textwidgetClass</class>
 <widget class="QWidget" name="textwidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>textwidget</string>
  </property>
  <widget class="QPushButton" name="pushButtontext1">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>470</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtontext2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>470</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="Edittext1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>671</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
